nominal/core/client.py

```diff
@@ -376,24 +376,37 @@ def create_dataset_from_io(
 
         file_type = FileType(*file_type)
         s3_path = upload_multipart_io(self._clients.auth_header, dataset, name, file_type, self._clients.upload)
-        request = ingest_api.TriggerIngest(
-            labels=list(labels),
-            properties={} if properties is None else dict(properties),
-            source=ingest_api.IngestSource(s3=ingest_api.S3IngestSource(path=s3_path)),
-            channel_config=(
-                None
-                if prefix_tree_delimiter is None
-                else ingest_api.ChannelConfig(prefix_tree_delimiter=prefix_tree_delimiter)
-            ),
-            dataset_description=description,
-            dataset_name=name,
-            timestamp_metadata=ingest_api.TimestampMetadata(
-                series_name=timestamp_column,
-                timestamp_type=_to_typed_timestamp_type(timestamp_type)._to_conjure_ingest_api(),
-            ),
+        request = ingest_api.IngestRequest(
+            options=ingest_api.IngestOptions(
+                csv=ingest_api.CsvOpts(
+                    source=ingest_api.IngestSource(s3=ingest_api.S3IngestSource(path=s3_path)),
+                    target=ingest_api.DatasetIngestTarget(
+                        new=ingest_api.NewDatasetIngestDestination(
+                            labels=list(labels),
+                            properties={} if properties is None else dict(properties),
+                            channel_config=(
+                                None
+                                if prefix_tree_delimiter is None
+                                else ingest_api.ChannelConfig(prefix_tree_delimiter=prefix_tree_delimiter)
+                            ),
+                            dataset_description=description,
+                            dataset_name=name,
+                        )
+                    ),
+                    timestamp_metadata=ingest_api.TimestampMetadata(
+                        series_name=timestamp_column,
+                        timestamp_type=_to_typed_timestamp_type(timestamp_type)._to_conjure_ingest_api(),
+                    ),
+                    additional_file_tags=None,
+                    channel_prefix=prefix_tree_delimiter,
```

Comment:
this is a bug - this is not the delimiter but a string to prefix all channel names by!